modules/resolving-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,67 @@
+* You have added a custom model-serving runtime for TGIS or have created a copy of the pre-installed standalone TGIS runtime. For more information, see link:{odhdocshome}/serving_models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {oai-admin-group}) in {openshift-platform}.
+* You have installed the single-model serving platform. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#about-the-single-model-serving-platform_serving-large-models[About the single-model serving platform]
+* You have added a custom model-serving runtime for TGIS or have created a copy of the pre-installed standalone TGIS runtime. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+
+.Procedure
+. From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
++
+The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled.
++
+. Find your custom model-serving runtime, click the action menu (&#8942;) next to the runtime and select *Edit*.
++
+The embedded YAML editor opens and shows the contents of the custom model-serving runtime.
++
+. Add or update the `BATCH_SAFETY_MARGIN` environment variable and set the value to 30. Similarly, add or update the `ESTIMATE_MEMORY_BATCH_SIZE` environment variable and set the value to 8.
++
+[source]
+----
+spec:
+  containers:
+    env:
+    - name: BATCH_SAFETY_MARGIN
+      value: 30
+    - name: ESTIMATE_MEMORY_BATCH
+      value: 8
+----
++
+[NOTE]
+====
+The `BATCH_SAFETY_MARGIN` parameter sets a percentage of free GPU memory to hold back as a safety margin to avoid OOM conditions. The default value of `BATCH_SAFETY_MARGIN` is `20`. The `ESTIMATE_MEMORY_BATCH_SIZE` parameter sets the batch size used in the memory auto-tuning algorithm. The default value of `ESTIMATE_MEMORY_BATCH_SIZE`  is `16`.
+====
+. Click *Update*.
++ 
+The *Serving runtimes* page opens and shows the list of runtimes that are installed. Observe that the custom model-serving runtime you updated is shown.
++
+. You must redeploy the impacted model for the updated or added parameters in the model-serving runtime to take effect:
+.. From the {productname-short} dashboard, click *Model Serving* > *Deployed Models*.
+.. Find the model you want to update, click the action menu (⋮) beside the model, and click *Delete*.
```

Comment:
A couple of suggestions here:

- "Find the model **that** you want to **redeploy**..."
- "click the action menu (&#8942;) next to the model..."

- click the action menu (&#8942;) next to the model...